src/com/goide/gotemplate/parser/GoTemplateParserUtil.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2013-2015 Sergey Ignatov, Alexander Zolotov, Florin Patan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.goide.gotemplate.parser;
+
+import com.intellij.lang.parser.GeneratedParserUtilBase;
+
+/**
+ * Created by josesantos on 21/12/15.
+ */
+public class GoTemplateParserUtil extends GeneratedParserUtilBase {
```

Comment:
It's possible to use a basic `com.intellij.lang.parser.GeneratedParserUtilBase` as a parser util class in the bnf file.